src/main/java/bit/javaoop/FixedBudgetManager.java

```diff
@@ -10,20 +13,18 @@
     }
 
     public FixedBudgetManager(String name, double salary) {
-        this(name, salary, 20000);
+        this(name, salary, BUDGET_INIT);
     }
 
     @Override
     public boolean isSatisfied() {
-
-        return getSalary() > 20000 && getBudget() < 15000;
+        return (getSalary() > MIN_OK_SALARY) && (getBudget() < MAX_OK_BUDGET);
     }
 
     @Override
     public boolean canHireEmployee(Employee newEmployee) {
         double sum = newEmployee.getSalary();
-        for (Employee temp : getEmployees()
-                ) {
+        for (Employee temp : getEmployees()) {
```

Comment:
`for (Employee e : getEmployees())`
raczej używamy w foreachach szczególnie takich krótkich jednoliterowych nazw no i najlepiej żeby ta nazwa mówiła nam cokolwiek bo taki tmp nie mówi nic:)